pkg/asset/ignition/bootstrap/bootstrap_test.go

```diff
@@ -0,0 +1,67 @@
+package bootstrap
+
+import (
+	"encoding/json"
+	"strings"
+	"testing"
+
+	igntypes "github.com/coreos/ignition/config/v2_2/types"
+	"github.com/golang/mock/gomock"
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/mock"
+)
+
+func TestBootstrapLoad(t *testing.T) {
+	tests := []struct {
+		name       string
+		config     *igntypes.Config
+		wantFound  bool
+		wantErrMsg string
+	}{
```

Comment:
also the common case for load that we would like to validate is user's adding a a new file..
can we add some unit tests that help us ensure we are valid for those use-cases?

Reply:
ok, i'll give that a go. thanks for the review!